File: Scala0821/src/main/scala/com/lfw/unit7/scalapackage/TestTiger.scala
package com.lfw.unit7.scalapackage

object TestTiger {
  def main(args: Array[String]): Unit = {
    //使用 xh 的 Tiger
    val tiger1 = new com.lfw.unit7.scalapackage.xh.Tiger
    //使用 xm 的 Tiger
    val tiger2 = new com.lfw.unit7.scalapackage.xm.Tiger
    println(tiger1)
    println(tiger2)
  }
}

File: Scala0821/src/main/scala/com/lfw/unit7/javapackage/TestTiger.java
package com.lfw.unit7.javapackage;

public class TestTiger {
    public static void main(String[] args) {
        //使用 xm 的 Tiger
        com.lfw.unit7.javapackage.xm.Tiger tiger01 = new com.lfw.unit7.javapackage.xm.Tiger();
        com.lfw.unit7.javapackage.xh.Tiger tiger02 = new com.lfw.unit7.javapackage.xh.Tiger();
        System.out.println("tiger01=" + tiger01);
        System.out.println("tiger02=" + tiger02);
    }
}
